src/core/reference/include/openvino/reference/slice_scatter.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/shape.hpp"
+#include "openvino/core/type/element_type.hpp"
+
+namespace ov {
+namespace reference {
+template <class T>
+void slice_scatter(const T* data,
+                   const Shape& data_shape,
+                   const T* updates,
+                   const Shape& updates_shape,
+                   T* out,
+                   size_t elem_size,
+                   const std::vector<int64_t>& starts,
+                   const std::vector<int64_t>& steps,
+                   const std::vector<int64_t>& axes) {
```

Comment:
The `axes` could be ov::AxisSet or ov::AxisVector then then normalization will be required.